features/step_definitions/search_functionality_steps.rb

```diff
@@ -0,0 +1,23 @@
+Given(/^that i am on the landing page$/) do
+  visit root_path
+end
+
+Given(/^there is a field "([^"]*)"$/) do |field|
+  find_field(field).value
```

Comment:
@faraznaeem @satorinic  Instead of finding the field and getting its value (_which you don't use for anything else_) what you probably want to do here is have an assertion that the field is actually present on the page.